src/components/TileCatalogNew/TileCatalogNew.jsx

```diff
@@ -147,14 +147,27 @@ const TileCatalogNew = ({
               ) : null}
             </div>
           </div>
+          {tiles && tiles.length && !error ? (
+            <div className={`${iotPrefix}--tile-catalog--tile-canvas--content`}>{renderGrid()}</div>
+          ) : (
+            <Tile className={`${iotPrefix}--tile-catalog--empty-tile`}>
+              {error || (
+                <>
+                  <Bee32 />
+                  <p>{i18n.error}</p>
```

Comment:
Since this is only shown when theres no error, should this be changed to describe why it got to this state? It seems this would only appear if tiles is empty or null so a better message would describe that empty state